mypy/build.py

```diff
@@ -2841,20 +2847,21 @@ def load_graph(sources: List[BuildSource], manager: BuildManager,
         # - In this case A's cached *direct* dependencies are still valid
         #   (since direct dependencies reflect the imports found in the source)
         #   but A's cached *indirect* dependency on C is wrong.
-        dependencies = [dep for dep in st.dependencies if st.priorities.get(dep) != PRI_INDIRECT]
+        dependencies = [dep for dep in st.dependencies if st.priorities.get(dep) != PRI_INDIRECT] \
+            if st.dependencies else []
```

Comment:
This is not needed. If `st.dependencies` is empty, then the list comprehension will also be empty.

Reply:
`st.dependencies` could be None though and that fails